common/run_as_userid.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+parent: Common
+---
+
+# Run as UserID
+
+Run pyRACF commands as another UserID.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .experimental }
+> _This functionality is **Experimental** and is subject to major changes and even being removed entirely._
+
+&nbsp;
+
+{: .note}
+> _Changes made using the functionality described here are scoped to the target "Admin" object instance._
+
+&nbsp;
+
+pyRACF can now leverage a feature of IRRSMO00 where commands can be run under a specified userid. In order to take advantage of this feature, the caller must have `UPDATE` access to the `userid.IRRSMO00` resource in the `SURROGAT` class, where userid represents the specific userid you wish to execute commands as. Further information can be found outlined in [our dependencies note](../../index).
+
+## Example
+
+###### Python Script
+```python 
+from pyracf import UserAdmin
+
+user_admin = UserAdmin(run_as_userid="squidwrd")
+# Any future commands would be run under squidwrd's authority

```

Comment:
```python
# all subsequent requests will be made using squidwrd's authority.
```

Reply:
agree, changed, but kept the "all" as "All".